profile-patches/ibm-cloud-managed/06-operator.yaml-patch

```diff
@@ -1,6 +1,7 @@
-- op: replace
-  path: /metadata/annotations
-  value:
-    include.release.openshift.io/ibm-cloud-managed: "true"
+- op: remove
+  path: /metadata/annotations/include.release.openshift.io~1self-managed-high-availability
+- op: add
+  path: /metadata/annotations/include.release.openshift.io~1ibm-cloud-managed
+  value: "true"
```

Comment:
Can you elaborate on what this is doing?

Reply:
From the 024493c9fd9c3e commit message:

> I'm also adjusting the JSON Patch file to remove [3] the
> self-managed-high-availability annotation and add [4] the
> ibm-cloud-managed annotation, instead of clobbering the whole
> annotation set, to preserve additional annotations like the capability
> name.  '~1` is the JSON Pointer escape for '/' [5].
> ...
> [3]: https://datatracker.ietf.org/doc/html/rfc6902#section-4.2
> [4]: https://datatracker.ietf.org/doc/html/rfc6902#section-4.1
> [5]: https://datatracker.ietf.org/doc/html/rfc6901#section-3